webview/src/plots/components/App.test.tsx

```diff
@@ -1108,7 +1147,11 @@ describe('App', () => {
 
         beforeEach(async () => {
           // eslint-disable-next-line testing-library/no-render-in-setup
```

Comment:
[C] If we're ok with this we should disable the rule globally.

Reply:
Done